### ISSUE TITLE
[Feature Request] Capture achievement and mastery notifications with automatic screenshot

### ISSUE THREAD

**Issue by AuburnRDM** [libretro/RetroArch#14032]
Currently, RetroArch's automatic screenshot capture upon earning an achievement doesn't capture anything but the game source itself.

However, if the screenshot button is pressed manually, RetroArch is able to capture any overlays on the screen.
![Golden Sun (USA, Europe)-220608-015257](https://user-images.githubusercontent.com/107295653/173176067-817aacf5-3685-442d-a7fc-5c293a68b23e.png)

Would it be possible to integrate this full screenshot capability to the automatic screenshot feature? Thanks!

**Comment by Jamiras** [comment]
This is essentially a duplicate of #11364, with the additional request to support mastery notifications.

**Comment by MestreLion** [comment]
Yes, please! Not only mastery notifications, but everything on the screen, just like standalone (GPU-enabled) screenshots already are.

It's very frustrating that my most important screenshots (the achievement ones) are 240 x 224 with no shaders or notifications, while my ordinary screenshots are 1920x1080 with full xBRZ glory.

**Comment by MrROBINKONG** [comment]
I whould also like for this to be a thing😁
